### DISCUSSION TITLE
How to get first steel?

### DISCUSSION THREAD

**Discussion by SPAZMASTIC** [TechReborn/TechReborn#2553] [Q&A]
Trying to do TechReborn as a standalone mod, and I'm having trouble figuring out how to make steel. You need a blast furnace to make the machine casings, but you need machine casings to make the blast furnace. Is there something I'm missing?

**Comment by xanthian** [comment]
You don't need a blast furnace for machine casings.
They use Iron / Refined Iron / Aluminium
